.github/workflows/code-coverage.yml

```
@@ -0,0 +1 @@
+Hello World! 
```

actionlint (1.7.12)

[error] "jobs" section is missing in workflow

(syntax-check)